CMakeLists.txt

```diff
@@ -67,14 +68,35 @@ function(copy_spout_file)
 	add_custom_command(
 		TARGET win-spout POST_BUILD
 		COMMAND ${CMAKE_COMMAND} -E copy
-			"${CMAKE_CURRENT_SOURCE_DIR}/deps/spout/VS2012/Binaries/${_src_suffix}/Spout.dll"
+			"${CMAKE_CURRENT_SOURCE_DIR}/deps/Spout2/BUILD/Binaries/${_src_suffix}/Spout.dll"
 			"${OBS_OUTPUT_DIR}/$<CONFIGURATION>/${OBS_PLUGIN_DESTINATION}/Spout.dll")
 	add_custom_command(
 		TARGET win-spout POST_BUILD
 		COMMAND ${CMAKE_COMMAND} -E copy
-			"${CMAKE_CURRENT_SOURCE_DIR}/deps/spout/SPOUT_LIBRARY/Binaries/${_src_suffix}/SpoutLibrary.dll"
+			"${CMAKE_CURRENT_SOURCE_DIR}/deps/Spout2/BUILD/Binaries/${_src_suffix}/Spout.pdb"
+			"${OBS_OUTPUT_DIR}/$<CONFIGURATION>/${OBS_PLUGIN_DESTINATION}/Spout.pdb")
+	add_custom_command(
+		TARGET win-spout POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy
+			"${CMAKE_CURRENT_SOURCE_DIR}/deps/Spout2/BUILD/Binaries/${_src_suffix}/SpoutLibrary.dll"
 			"${OBS_OUTPUT_DIR}/$<CONFIGURATION>/${OBS_PLUGIN_DESTINATION}/SpoutLibrary.dll")
+	add_custom_command(
+		TARGET win-spout POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy
+			"${CMAKE_CURRENT_SOURCE_DIR}/deps/Spout2/BUILD/Binaries/${_src_suffix}/SpoutLibrary.pdb"
```

Comment:
Worried that this will break when there are no `pdb` files (ie when only built for release)

Reply:
Update: It omits pdb's for "Release" now.
1. Should we add something that checks if pdb's are available in "Debug" - else build without
2. or force fail if config is "Debug" and there are no pdb's ?